config/config.exs

```diff
@@ -12,9 +12,9 @@ config :absinthe, :schema, Accent.GraphQL.Schema
 # Configures the endpoint
 config :accent, Accent.Endpoint,
   root: Path.expand("..", __DIR__),
-  http: [port: System.get_env("PORT")],
-  url: [host: System.get_env("CANONICAL_HOST") || "localhost"],
-  secret_key_base: System.get_env("SECRET_KEY_BASE"),
+  http: [port: "${PORT}"],
```

Comment:
Why use the `”${ENV}”` syntax alone with the limitation it creates? Especially since distillery 2.0 added clean support for`System.get_env/1`.

btw, in to previous recipe, you can use `System.get_env(“ENV”) || “${ENV}”` to stay compatible with `mix` for development.